src/node/hooks/express/importexport.js

```diff
@@ -33,13 +38,38 @@ exports.expressCreateServer = function (hook_name, args, cb) {
   });
 
   // handle import requests
+  args.app.use('/p/:pad/import', limiter);
   args.app.post('/p/:pad/import', async function(req, res, next) {
     if (await hasPadAccess(req, res)) {
       let exists = await padManager.doesPadExists(req.params.pad);
       if (!exists) {
+        console.warn("Someone tried to import to a pad that doesn't exist");
```

Comment:
I'll always try to add infos about the actual parameter values in the log messages. Rule of thumb is: a log message should be a `` `template string with some ${variable} value inside` `` instead of `"a simple fixed string"`.

This is more a operator rather than end-user UX, but I think it can improve ergonomics.
For example in this case we could add the non-existend pad name.

I'll do it here and in similar places, no problem.

Reply:
yeah good point, will do so in future :)

Reply:
Done by me for this PR.